app/assets/stylesheets/mixins/_forms.scss

```diff
@@ -124,5 +124,17 @@
       }
     }
   }
-}
 
+  // newsletter signup form
+  #newsletter_signup {
+    background-color: #1F2A3D;
+    color: #FFFFFF;
+    border-color: #1222DD;
+    border-radius: 4px;
+    padding: 20px;
+    input[type=submit] {
+      background-color: #F40B0B;
+      color: #FFFFFF
+    }
+  }
+}
```

Comment:
Not sure if this was the right file to add this code. Looking forward to suggestions, I'm happy to move it.

Reply:
I think it would be best to add a file specifically for these styles. These don't really look like _form_ specific styles that we might want to use elsewhere.